### ISSUE TITLE
Simple Search Tab Re-Design

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-frontend#417]
**Problem Description:**
LUX would like to redesign the front end layout based on POW's mockups. This issue will support the re-design of the front end layout (#270) and specifically focus on the simple search tab re-design. 

Prerequisite: https://github.com/project-lux/lux-frontend/issues/416

**Expected Behavior/Solution:**
LUX simple search tabs to be re-configured to match the requirements and mockups below. 

**Requirements:**
- Tabs will be connected to the new merged facet and result section. 
- Tabs will contain their respective icon to the right side. 
- Icons will always appear for all scope tabs. 
- - `X results` will be placed underneath the <Scope Label> 

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] Wireframe/Mockup - Mike
- [ ] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [ ] Frontend requirements- TBD
- [ ] Are new regression tests required for QA - Amy
- [ ] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**
- https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22andy%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22warhol%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&ip=3&qt=objects&view=list&is=anySortName%3Adesc&sq=&fromAdvanced=true

**Dependencies/Blocks:**
- Perquisite: https://github.com/project-lux/lux-frontend/issues/416

**Related Github Issues:**
- https://github.com/project-lux/lux-frontend/issues/270
- https://github.com/project-lux/lux-frontend/issues/418

**Related links:**
[Presentation](https://www.figma.com/proto/bWS2orxFECNrG92HfG48BL/LUX-Yale-Design-Walkthrough?page-id=3377%3A9485&node-id=3377-9572&viewport=4999%2C493%2C0.26&t=Axk0QSGESY3qeZh1-1&scaling=scale-down&content-scaling=fixed)
[Design walkthrough](https://www.figma.com/design/bWS2orxFECNrG92HfG48BL/LUX-Yale-Design-Walkthrough?node-id=30-6364&node-type=canvas&t=Sy4U7OxDEOrthtNY-0)
[Prototype](https://www.figma.com/proto/bWS2orxFECNrG92HfG48BL/LUX-Yale-Design-Walkthrough?page-id=182%3A6364&node-id=3317-20738&viewport=-5411%2C-5010%2C0.36&t=yi4XGHaZfpsKRsqT-9&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=3317%3A20738&show-proto-sidebar=1)

**Wireframe/Mockup:**
Current: 
![image](https://github.com/user-attachments/assets/ad9fcd27-9bea-4c04-ae3d-686eac5d5917)

Expected: 
![image](https://github.com/user-attachments/assets/0f2eeeeb-a84f-4dba-b3aa-545c2e96f9d2)

**Comment by prowns** [comment]
@kamerynB - can you validate the grays used in the tabs? It seems to be a much darker gray than I'm seeing deployed.

Also - are you planning to add the Object/Works (?) overlay as part of this ticket, or is that separate?

**Comment by jffcamp** [comment]
@kamerynB , On the Concept Tab
https://lux-front-tst.collections.yale.edu/view/results/concepts?q=%7B%22text%22%3A%22print-making%22%2C%22_lang%22%3A%22en%22%7D&sq=print-making

The results read,, "2757 Objects results" Should be "2757 Concepts results"

**Comment by roamye** [comment]
1. The bottom end of the tab can be seen when it connects to the results page. Not a major issue, but would be nice if the line came from the results page rather than the tab. 
![image](https://github.com/user-attachments/assets/bf884fa3-9a14-4db3-b204-614e181a00d8)
2. Jeff's comment above where the concept tab reads as `object results` rather than `concepts results` 
3. (minor issue) | the people and groups icon, as shown above is inconsistent with where the other icons are placed. This also affects the "66 results" as it is placed in the middle.

**Comment by roamye** [comment]
Approved by UAT but requires hot fix for:
 - concepts tab has the wrong label and description for the search results. 

New ticket will be needed to address: 
 - the addition of the (?) icon and overlay. 
- the grey color. 
- the curving of the corner within the tab that connects to the search results.
- people and groups tab spacing for "- results" needs to be further left.

**Comment by roamye** [comment]
Closing, Looks good in PROD. 

New issue for the curvature has been made: https://github.com/project-lux/lux-frontend/issues/479
all other hot fixes were addressed already in this ticket.
